releasenotes/notes/gate-documentation-5a376c8a471a13e1.yaml

```diff
@@ -0,0 +1,7 @@
+---
+prelude: >
```

Comment:
This should be a new feature section or we should save it for the release. The prelude section is special in that we can only have one for a release since it doesn't really combine multiple elements well since it's a free form paragraph section and not a list. Typically before a release I've been writing a small blurb summarizing the big changes for the release.  This release is turning out to be quite large (not by patch count but because of several substantial changes) so we'll probably have to collaborate on a good prelude for the release.

Reply:
yeah i put it at the top here so we don't forget to write a prelude about documentation that came out of the docathon. i figured we can tweak it towards the release. but documentation doesn't quite fit a "feature" in my mind